frontend/src/app/projects/models/model-detail/model-detail.component.ts

```diff
@@ -5,22 +5,24 @@
 
 import { Component, OnDestroy, OnInit } from '@angular/core';
 import { BehaviorSubject, filter, Subscription } from 'rxjs';
-import { GitModelService } from 'src/app/projects/project-detail/model-overview/model-detail/git-model.service';
 import { ModelService } from 'src/app/services/model/model.service';
 import {
   T4CModel,
-  T4cModelService,
+  T4CModelService,
 } from 'src/app/services/modelsources/t4c-model/t4c-model.service';
 import { ProjectService } from 'src/app/services/project/project.service';
-import { GitModel } from 'src/app/services/source/source.service';
+import {
+  GetGitModel,
+  GitModelService,
+} from '../../project-detail/model-overview/model-detail/git-model.service';
```

Comment:
Same here

```suggestion
} from 'src/app/projects/project-detail/model-overview/model-detail/git-model.service';
```

Reply:
Done. There are many references left (many of them unrelated to my PR). I think we should handle them in another PR. I created the issue for it (see last comment)

Reply:
That’s why I stopped reporting them at some point ^^